### ISSUE TITLE
Per Template Ignore Checks

### ISSUE THREAD

**Issue by danquack** [aws-cloudformation/cfn-python-lint#1157]
*cfn-lint version: (`cfn-lint --version`)*
cfn-lint 0.22.3
*Description of issue.*
On a given template it would be nice to either inline, or per file ignore certain checks.

Please provide as much information as possible:
* Feature request:
Sometimes particular files we'd like to ignore certain linting violations, but not cross cut. Allow for per template ignores.

**Comment by kddejong** [comment]
Are you wanting something beyond this?
https://github.com/aws-cloudformation/cfn-python-lint#template-based-metadata
or
https://github.com/aws-cloudformation/cfn-python-lint#resource-based-metadata

**Comment by danquack** [comment]
Yes, I have a particular file that I want to ignore a single check. I am unable to fix it, due to a production outage needed on update. I have a similar template for another environment, that I want to validate, that is right. So I can't do a resource based ignore, because it would ignore it on the other template. For now, I have to ignore the whole file for the way it's written today.

**Comment by kddejong** [comment]
The Metadata configuration in a file will only ignore that rule for that template.

file test1.yaml
```yaml
AWSTemplateFormatVersion: "2010-09-09"
Metadata:
  cfn-lint:
    config:
      ignore_checks:
        - E3001
Resources:
  ExampleResource:
    Type: AWS::Bad::Resource
    Properties: {}
```

file test2.yaml
```yaml
AWSTemplateFormatVersion: "2010-09-09"
Resources:
  ExampleResource:
    Type: AWS::Bad::Resource
    Properties: {}
```

if you run
```
cfn-lint test1.yaml test2.yaml
E3001 Invalid or unsupported Type AWS::Bad::Resource for resource ExampleResource in us-east-1
test2.yaml:4:5

```

**Comment by danquack** [comment]
Ohhh, this is what I wanted. User error! Thanks for the clarification

**Comment by kddejong** [comment]
You're welcome
